src/engine/renderer/tr_shade.cpp

```diff
@@ -48,6 +48,16 @@ static void EnableAvailableFeatures()
 				glConfig2.realtimeLighting = false;
 			}
 
+			if ( !glConfig2.uniformBufferObjectAvailable ) {
```

Comment:
These could be else-if's

Reply:
Perhaps, I just did the same thing as the original code.

Reply:
I prefer to have twice the warnings.